EmpleoDotNet.Core/Domain/JobOpportunity.cs

```diff
@@ -22,11 +22,21 @@ public class JobOpportunity : EntityBase
         public int? LocationId { get; set; }
 
         /// <summary>
-        /// Descripción de los requerimientos necesarios para aplicar al empleo
+        /// Descripción general del puesto
         /// </summary>
         public string Description { get; set; }
 
         /// <summary>
+        /// Descripción de los requisitos necesarios para aplicar al empleo
```

Comment:
@claudiosanchez should we do this in another issue?, Because not only the new properties have the documentation in a different language .